addons/mail/controllers/discuss.py

```diff
@@ -230,7 +230,7 @@ def discuss_starred_messages(self, max_id=None, min_id=None, limit=30, **kwargs)
     # --------------------------------------------------------------------------
 
     def _get_allowed_message_post_params(self):
-        return {'attachment_ids', 'body', 'message_type', 'partner_ids', 'subtype_xmlid', 'parent_id'}
+        return {'attachment_ids', 'body', 'link_preview_ids', 'message_type', 'partner_ids', 'subtype_xmlid', 'parent_id'}
```

Comment:
can't blindly trust user input here, the methods that make use of this need to validate that the provided ids are actually matching previews that are from the current user (and not any existing one) / related to that message in a way (possibly also validating in python that at least the url are present on the message)